packages/client/src/components/GrantsTable.vue

```diff
@@ -247,13 +246,19 @@ export default {
         this.loading = false;
       }
     },
-    sortingChanged(ctx) {
-      if (!ctx.sortBy) {
-        this.orderBy = '';
-      } else {
-        this.orderBy = `${ctx.sortBy}|${ctx.sortDesc ? 'desc' : 'asc'}`;
+    getAwardFloor(grant) {
+      let body;
+      try {
+        body = JSON.parse(grant.raw_body);
+      } catch (err) {
+        // Some seeded test data has invalid JSON in raw_body field
+        return undefined;
+      }
+      const floor = parseInt(body.synopsis && body.synopsis.awardFloor, 10);
+      if (Number.isNaN(floor)) {
+        return undefined;
```

Comment:
if `undefined` is returned can the UI handle that or will it break?

Reply:
The UI handles it fine, it just doesn't display any value which is correct because if this function returns undefined then there is no award floor